### DISCUSSION TITLE
Possible to get the x-amzn-requestid from DynamoDB response ?

### DISCUSSION THREAD

**Discussion by chamorin** [dynamoose/dynamoose#1394] [Q&A]
I was wondering if I can have the `x-amzn-requestid` from a request response. In my case I'm doing a `Model.get(id)` and I want to add the AWS request id in my error body response. So if the request does not found the resource, my API would return a 404 with the AWS request id from the request that did the `get()`.

**Comment by fishcharlie** [comment]
https://github.com/dynamoose/dynamoose/issues/1395
